### DISCUSSION TITLE
Is it possible to draw a chart like attached image?

### DISCUSSION THREAD

**Discussion by feents** [nhn/tui.chart#674] [Q&A]
Is it possible to draw chart like attached image?

I want to group categories like red box.

![chart](https://user-images.githubusercontent.com/67404954/119102223-c2041a00-ba54-11eb-8940-61dbcd93d95b.png)

**Comment by jung-han** [comment]
We do not currently offer that option. It seems that the function can be developed after the map chart is developed.
